datatypes/src/plots/multi_line_plot.rs

```diff
@@ -0,0 +1,110 @@
+use crate::plots::{Plot, PlotData, PlotMetaData};
+use crate::primitives::{Measurement, TimeInstance};
+use crate::util::Result;
+
+pub struct DataPoint {
+    pub series: String,
+    pub time: TimeInstance,
+    pub value: f64,
+}
+
+impl From<(String, TimeInstance, f64)> for DataPoint {
+    fn from(p: (String, TimeInstance, f64)) -> Self {
+        Self {
+            series: p.0,
+            time: p.1,
+            value: p.2,
+        }
+    }
+}
+
+pub struct MultiLineChart {
```

Comment:
Please briefly describe what this plot is about.

Reply:
1de04c56901a6259b4f40ce4bac73c21a1fe4b38